wiki/Developer-Documentation.md

```diff
@@ -125,6 +125,33 @@ and performs various checks, it may take a couple of minutes to complete. If you
 are running this from a partition that is nearly full, the test may fail. Please
 refer to `maxSpaceUtilizationThreshold` in [[Configuration Reference]].
 
+#### Writing Tests
```

Comment:
Yeah, the long term plan is that `make gtest` is temporary until we transition everything to Google Test. Then `make test` will point to Google Test and qfstest.sh will be no more.